docker-compose.yaml

```diff
@@ -59,7 +59,8 @@ services:
     init: true # xvfb-run hang fix?
     tty: true
 
-  torch_agents: &torch_agents
+  torch_agents:
+    &torch_agents
```

Comment:
hum, does it work ?

Reply:
It does, and the autoformatter suggests it

Reply:
mmh ok